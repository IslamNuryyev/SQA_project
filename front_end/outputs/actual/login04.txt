***************************************************************
Welcome to the Auction App
 -- Created by     -- 
 -- Luis Octavo    -- 
 -- Owais Najmi    -- 
 -- Islam Nuryyev  -- 
 -- Bhargav Parekh -- 
***************************************************************

Enter command: 
Enter username: 
User does not exist

Enter command: 
Logout successful!
