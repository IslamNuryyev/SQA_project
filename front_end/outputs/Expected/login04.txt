***************************************************************
Welcome to the Auction App
 -- Created by     -- 
 -- Luis Octavo    -- 
 -- Owais Najmi    -- 
 -- Islam Nuryyev  -- 
 -- Bhargav Parekh -- 
***************************************************************

Enter command: 
Enter username: 
LT04ERR- User does not exist

Enter command: 
Logout successful!
